local/example.env

```diff
@@ -1,13 +1,12 @@
-# these variables are required by docker-compose.yml, fill with any strings
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
 POSTGRES_DB=postgres
-
-# these variables are required by application
-DATABASE_URL=postgresql://postgres:postgres@localhost:5432/postgres
-PGADMIN_DEFAULT_EMAIL=user@example.com
-PGADMIN_DEFAULT_PASSWORD=x
+POSTGRES_HOST=db
```

Comment:
Why is it required to update example.env? I feel that README instructions may not work with these default values. E. g. "POSTGES_HOST" should be either `postgres` (the name of docker-compose service) or `localhost` (in case of running backend outside of docker-compose machinery).

Reply:
I saw .env file in telegram, and without changing it to old "example.env" it didn't work. Before I did by your README, it worked, but didn't show me bugs(and bugs were 100%), but after I changed "example.env" I saw bug, @anton-serozhechkin show me
Maybe I did something wrong, but after that it just worked

Reply:
Now I tried it with old "example.env", and it works. Maybe I did something wrong before, and it didn't work. So probably, I wouldn't have change file, if it worked that time.
But now it works, so I use the old "example.env"